.devcontainer/devcontainer.json

```diff
@@ -1,16 +1,36 @@
 {
   "name": "Discourse",
-  "image": "discourse/discourse_dev:release",
-  "workspaceMount": "source=${localWorkspaceFolder}/../..,target=/var/www/discourse,type=bind",
-  "workspaceFolder": "/var/www/discourse",
-  "settings": {
-    "search.followSymlinks": false
-  },
-  "postStartCommand": "sudo /sbin/boot",
-  "extensions": ["rebornix.Ruby"],
-  "forwardPorts": [9292],
+  "image": "docker.io/discourse/discourse_dev:20241111-0710",
```

Comment:
We'll need to update the image regularly?

Reply:
Yes